doc/metrics.md

```diff
@@ -3,24 +3,24 @@ This document aims to help users that are not familiar with metrics exposed by t
 All metrics documented here are auto-generated by the utility tool `tools/metricsdocs` and reflects exactly what is being exposed.
 
 ## Containerized Data Importer Metrics List
-### clone_progress
+### kubevirt_cdi_clone_pods_high_restart
+CDI cloner pods with high restart count. Type: Gauge.
```

Comment:
CDI cloner pods with high restart count ->
The number of CDI clone pods that have a high restart count

Reply:
+1 Fixed for all